test/presenters/taxon_presenter_test.rb

```diff
@@ -75,13 +75,27 @@
     end
   end
 
+  describe 'supergroup_sections' do
+    it 'returns a list of supergroup details' do
+      section_keys = %i(show_section title documents)
+
+      taxon = mock
+      taxon.stubs(:guidance_and_regulation_content).returns([])
+      taxon_presenter = TaxonPresenter.new(taxon)
+
+      taxon_presenter.sections.each do |section|
+        assert_equal(section.keys.sort, section_keys.sort)
```

Comment:
I think it would be helpful to make the expectation here more explicit, in case for whatever reason `section.keys.sort` gets messed up and is returning the wrong thing.

Reply:
@emmabeynon What do you mean by making the expectation more explicit?

Reply:
As a person who's not familiar with the code, it would be helpful to see exactly what `section.keys.sort` returns.  Just makes the test a bit easier to read and understand.

Reply:
@emmabeynon Thanks, that makes sense! 👍